### ISSUE TITLE
enable compression for load-redshift and s3-node

### ISSUE THREAD

**Issue by p5k6** [coursera/dataduct#238]
tl;dr - I would like to add compression options to load-redshift and s3_node. Relevant aws documentation:  [s3-node](http://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/dp-object-s3datanode.html#d0e14697) and [RedshiftCopyActivity](http://docs.aws.amazon.com/redshift/latest/dg/t_loading-gzip-compressed-data-files-from-S3.html) 

We have a use case at my employer where we have to push some fairly large tables (about 500 GB uncompressed) from mysql => redshift. I created a custom step (based on extract-rds) to compress throughout the pipeline. However, this required some mods to both s3-node and load-redshift. I wanted to pass these options back into the mainline project. PR forthcoming

Also - I'd be happy to contribute the custom step (I called it ExtractMysqlGzip, for lack of a better term). The only reason I did not create a PR for this is - well, the custom step is pretty hacky to get around aws's limitations imposed on s3datanodes that have compression enabled.

**Comment by p5k6** [comment]
FYI - [here's a link](https://gist.github.com/p5k6/9c8cf4c22d4aeb19bb62dd7e906d94f7) to the custom step I'm using to trick data pipeline into compressing the data coming out of mysql

**Comment by p5k6** [comment]
Merged in PR 239
